.github/workflows/kvrocks.yaml

```
@@ -272,6 +283,7 @@ jobs:
           sudo mkdir -p $RISCV_PATH
           wget "${{ matrix.riscv_toolchain_url }}" -O riscv-toolchain.tar.xz
           sudo tar -xvf riscv-toolchain.tar.xz -C $RISCV_PATH --strip-components=1
+          rm riscv-toolchain.tar.xz
```

```suggestion
          wget -qO- "${{ matrix.riscv_toolchain_url }}" | sudo tar -Jxv -C $RISCV_PATH --strip-components=1
```

Hi @PragmaTwice ,

Maybe this on-the-fly way would reduce the temp storage during wget and wouldn't leave the file in filesystem if the tarball is crashed. 😊

Best Regards,
Edward

---

Thank you. It looks good but maybe we can defer it as an enhancement.